### DISCUSSION TITLE
[Matrix Field] Group Block Types + Add possibility for tabs

### DISCUSSION THREAD

**Discussion by davidhellmann** [craftcms/cms#8500] [Ideas] [answered]
With Craft 3 I try to forget NEO and go back to Matrix cause it's core and more secure. 
But I think the field can become some enhancements to have a better UX at all.

1. Organize Block Types into Groups to be more organized (
![image](https://user-images.githubusercontent.com/764292/35918218-46665682-0c11-11e8-8051-fe3d17f3c7e9.png)
![image](https://user-images.githubusercontent.com/764292/35918226-4cdb6df4-0c11-11e8-9eab-1e8326e2b76f.png)

2. Split Content and Settings into Tabs when possible
![image](https://user-images.githubusercontent.com/764292/35918249-6ff9e7a2-0c11-11e8-8b8e-03bd2e126711.png)

Sure you know the Plugin Pimp My Matrix :)
https://github.com/supercool/Pimp-My-Matrix/ 

Say it again, that would be a huge enhancement :)

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.0.0-alpha.7 was just tagged, which adds multiple tab support to inline-editable Matrix blocks :tada:

<img src="https://github.com/craftcms/cms/assets/47792/b643df50-07df-4c39-8fe2-c94efcd17f0e" width="555" height="151" alt="A Matrix block with two tabs">

**Reply by davidhellmann** [reply]
Nice, great! Thanks!

**Comment by wmdhosting** [comment]
thats great :)
is there will be option Group Block Types  so we can stop use all plugins that greatly slow done entry page load..
